include/cutlass/gemm/collective/intel_pvc_mma.hpp

```diff
@@ -0,0 +1,353 @@
+
+  static constexpr int SubgroupSize = DispatchPolicy::SubgroupSize;
+  
+  static constexpr int DpasM = get<0>(shape(typename TiledMma::LayoutA_TV{})); // rows per dpas operation per sub_group for Matrix A
+  static constexpr int DpasN = get<1>(shape(typename TiledMma::LayoutB_TV{})); // cols per dpas operation per sub_group for Matrix B
+  static constexpr int DpasK = get<1>(shape(typename TiledMma::LayoutA_TV{})); // cols per dpas operation per sub_group for Matrix A
+
+  static constexpr int FragsM = get<0>(TileShape{}) / DpasM; // A frags per sub_group
+  static constexpr int FragsN = get<1>(TileShape{}) / DpasN; // B frags per sub_group
+
+  struct Arguments {
+    ElementA const* ptr_A;
+    StrideA dA;
+    ElementB const* ptr_B;
+    StrideB dB;
+  };
+
+  struct Params {
+    using XE_Copy_A = decltype(make_xe_2d_copy<GmemTiledCopyA>(make_tensor(static_cast<ElementA const*>(nullptr), 
+                                repeat_like(StrideA{}, int32_t(0)), StrideA{})));
+    using XE_Copy_B = decltype(make_xe_2d_copy<GmemTiledCopyB>(make_tensor(static_cast<ElementA const*>(nullptr), 
+                                repeat_like(StrideB{}, int32_t(0)), StrideB{})));
+    XE_Copy_A gmem_tiled_copy_a;
+    XE_Copy_B gmem_tiled_copy_b;
+  };
+
+  //
+  // Methods
+  //
+
+  CollectiveMma() = default;
+
+  template <class ProblemShape>
+  static constexpr Params
+  to_underlying_arguments(ProblemShape const& problem_shape, Arguments const& args, void* workspace) {
+    (void) workspace;
+
+    auto problem_shape_MNKL = append<4>(problem_shape, 1);
+    auto [M,N,K,L] = problem_shape_MNKL;
+
+    Tensor tensorA = make_tensor(args.ptr_A, make_layout(make_shape(M,K,L), args.dA));
+    Tensor tensorB = make_tensor(args.ptr_B, make_layout(make_shape(K,N,L), args.dB));
+
+    typename Params::XE_Copy_A copyA = make_xe_2d_copy<GmemTiledCopyA>(tensorA);
+    typename Params::XE_Copy_B copyB = make_xe_2d_copy<GmemTiledCopyB>(tensorB);
+    return Params{copyA, copyB};
+  }
+
+  /// Perform a subgroup-scoped matrix multiply-accumulate
+  template <
+    class FrgTensorD,
+    class TensorA,
+    class TensorB,
+    class FrgTensorC,
+    class KTileIterator,
+    class ResidueMNK
+  >
+  CUTLASS_DEVICE void
+  operator() (
+      FrgTensorD &accum,
+      TensorA gA,
+      TensorB gB,
+      FrgTensorC const &src_accum,
+      KTileIterator k_tile_iter, int k_tile_count,
+      ResidueMNK residue_mnk,
+      int thread_idx,
+      char *smem_buf,
+      Params const& mainloop) 
+  {
+    // static_assert(is_rmem<FrgTensorD>::value, "D tensor must be rmem resident.");
+    // static_assert(is_tuple<typename TensorA::engine_type::iterator::value_type>::value, "A tensor must be tuple iterator.");
+    // static_assert(is_tuple<typename TensorB::engine_type::iterator::value_type>::value, "B tensor must be tuple iterator.");
+    // static_assert(is_rmem<FrgTensorC>::value, "C tensor must be rmem resident.");
```

Comment:
Are they required? if so please enable them. otherwise please remove them

Reply:
This `operator()` function never gets used in the code as we use the `MainLoopPVCUnpredicated` dispatch policy only, we don't use the `MainloopIntelPVC` dispatch policy anywhere which is why this code doesn't get used yet.

Reply:
As I said if it is not used should be removed. If it is used it should be uncommented. You should not have commented code in your pr

Reply:
Should I remove the whole partial specialization or just the commented code?

Reply:
I am confused, why do you need to keep an unreachable code?